pkg/auth/service.go

```diff
@@ -260,8 +260,8 @@ func (s *DBAuthService) CreateUser(ctx context.Context, user *model.User) (int64
 		}
 		var id int64
 		err := tx.Get(&id,
-			`INSERT INTO auth_users (display_name, created_at, friendly_name, source) VALUES ($1, $2, $3, $4) RETURNING id`,
-			user.Username, user.CreatedAt, user.FriendlyName, user.Source)
+			`INSERT INTO auth_users (display_name, created_at, friendly_name, source, email) VALUES ($1, $2, $3, $4, $5) RETURNING id`,
+			user.Username, user.CreatedAt, user.FriendlyName, user.Source, user.Email)
```

Comment:
Please verify that is the table default value for email and if we are now setting null as email can be a problem, or if it is aligned to the value we insert for users without email.

Reply:
So both are true :). It is set as NULL and this is also the value for users without email